src/common/Network/Packet/PTPPacket.cpp

```diff
@@ -18,32 +18,38 @@ limitations under the License.
 
 void PTP::Header::dump(FILE *fd) {
     fprintf(fd, "\nPTP Header");
-    fprintf(fd, "\nMessageId 0x%.1X (%s), TransportSpecific 0x%.1X, PTP Version %d, MessageLength %d", trn_and_msg.msg_type_raw(),
-            trn_and_msg.msg_type_str(), trn_and_msg.trans_spec_raw(), *ver, *message_len);
+    fprintf(fd, "\nMessageId 0x%.1X (%s), TransportSpecific 0x%.1X, PTP Version %d, MessageLength %d",
+            trn_and_msg.msg_type_raw(), trn_and_msg.msg_type_str(), trn_and_msg.trans_spec_raw(), (int)*ver,
+            (int)*message_len);
```

Comment:
why cast everything to int?
static_cast, btw

Reply:
compiler threw errors when not casted

Reply:
switched to `static_cast<>`